packages/appkit-utils/src/CaipNetworkUtil.ts

```diff
@@ -209,14 +209,15 @@ export const CaipNetworksUtil = {
   getViemTransport(caipNetwork: CaipNetwork, projectId: string, customRpcUrls?: CustomRpcUrl[]) {
     const transports: HttpTransport[] = []
 
+    // Add custom RPC URLs
     customRpcUrls?.forEach(rpcUrl => {
       transports.push(http(rpcUrl.url, rpcUrl.config))
     })
 
+    // Add Reown RPC URL
     if (WC_HTTP_RPC_SUPPORTED_CHAINS.includes(caipNetwork.caipNetworkId)) {
```

Comment:
Also this is a static list prob we'd better to check out if there is an update to it (if new networks has been added or not)

Reply:
Tried to use blockchainapicontroller but it made all methods async and broke a bunch of flows :/. keeping as is for now